docs/src/main/asciidoc/_admin-guide/Galleon_provisioning.adoc

```diff
@@ -0,0 +1,207 @@
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to run a complete WildFly server. 
+By default, all standalone and domain configurations are installed.
+
+NB:
+
+* Append the release version to install an identified release, for example:
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current#16.0.0.Final --dir=my-wildfly-server
+----
+* Use _wildfly-core:current_ to only install a core server, for example:
+----
+galleon.sh install wildfly-core:current --dir=my-core-server
+----
+* Use _wildfly-servlet:current_ to only install a servlet server, for example:
+----
+galleon.sh install wildfly-servlet:current --dir=my-servlet-server
+----
+ 
+
+== Selecting the configurations to install
+
+Galleon allows to specify only a subset of the default configurations to be installed.
+
+To only install the standalone.xml configuration call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --default-configs=standalone/standalone.xml
+----
+
+To install multiple configurations, a comma separated list of default configurations can be provided, 
+for example:
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --default-configs=standalone/standalone.xml,domain/domain.xml
+----
+
+NB: A default configuration is identified by _<configuration model>_/_<configuration name>_. WildFly defines _standalone_ and _domain_ models. 
+The _configuration name_ is the XML configuration file name (eg: _standalone.xml_, _domain.xml_, _standalone-ha.xml_).
+
+== WildFly Galleon layers
+
+WildFly Galleon feature-packs expose some link:++https://docs.wildfly.org/galleon/#_layers++[Galleon layers]. 
+A layer identifies one or more server functionalities that can be installed on its own or 
+in combination with other layers. For example, if your application (eg: microservice) 
+is only making use of _jaxrs_ and _cdi_ server functionalities, you can choose to only install 
+the _jaxrs_ and _cdi_ layers. The _standalone.xml_ configuration would then only contain the 
+required subsystems and their dependencies.
+
+The benefit to install WildFly using Galleon layers, in addition to configuration 
+trim down, is that Galleon only install the needed content (JBOSS modules, scripts).
+
+To install a server only configured with jaxrs and cdi functionalities call:
+
+[source,options="nowrap"]
+----
+galleon.sh install wildfly:current --dir=my-wildfly-server --layers=cdi,jaxrs
+----
+
+Once installed, the directory _my-wildfly-server_ contains all that is expected to deploy an application
+that depends on _jaxrs_ and _cdi_. 
+
+NB:
+
+* Tools (jboss-cli, add-user, ...) are not installed when installing WildFly using layers. 
+Only server launcher scripts are installed into _bin_ directory.
```

Comment:
s/into/into the/g

Perhaps add a sentence -- "To include these tools include the _core-tools_ layer."